app/main/containers/Search/index.js

```diff
@@ -369,7 +369,7 @@ Search.propTypes = {
 function mapStateToProps(state) {
   return {
     selected: state.search.selected,
-    results: state.search.resultIds.map(id => state.search.resultsById[id]),
+    results: state.search.resultIds.map(id => state.search.resultsById[id]).sort((a, b) => b.selectCount - a.selectCount),
```

Comment:
I figured out the issue I was having when sorting only the basic apps. All apps start out with NaN as their selection count, so my sort wasn't working out. My pushed changes isolate the sorting to only basic apps.